### PR TITLE
Added Bianet dataset

### PR DESCRIPTION
**Pull request by param087**
Hi :hugs:, This is a PR for [Bianet: A parallel news corpus in Turkish, Kurdish and English; Source](http://opus.nlpl.eu/Bianet.php) dataset

### CODE REVIEW

File: datasets/bianet/bianet.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+Ataman, D. (2018) Bianet: A Parallel News Corpus in Turkish, Kurdish and English. In Proceedings of the LREC 2018 Workshop MLP-Moment. pp. 14-17. pdf"""
```
Comment (lhoestq) [inline datasets/bianet/bianet.py:25]: Can you use the bibtex format here ?

**Review by lhoestq — COMMENTED** [review]
Really cool !

I just left one comment about the citation

File: datasets/bianet/bianet.py (hunk 1 above)
Comment (param087) [reply datasets/bianet/bianet.py]: Hi @lhoestq, I have added the BibTeX and language tags.

File: datasets/bianet/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+languages:
+- en-to-ku
+  - en
+  - ku
+- en-to-tr
+  - en
+  - tr
+- ku-to-tr
+  - ku
+  - tr
```
Comment (lhoestq) [inline datasets/bianet/README.md:15]:
```suggestion
  en-to-ku
  - en
  - ku
  en-to-tr
  - en
  - tr
  ku-to-tr
  - ku
  - tr
```

File: datasets/bianet/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+licenses:
+- unknown
+multilinguality:
+- translation
+size_categories:
+- 1M<10M
```
Comment (lhoestq) [inline datasets/bianet/README.md:21]:
```suggestion
- n>1M
```

File: datasets/bianet/bianet.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                x = x.strip()
+                y = y.strip()
+                result = (
+                    sentence_counter,
+                    {
+                        "id": str(sentence_counter),
+                        "translation": {lang1: x, lang2: y},
+                    },
+                )
+                sentence_counter += 1
```
Comment (lhoestq) [inline datasets/bianet/bianet.py:149]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
Awesome thank :)

**Comment by lhoestq** [comment]
merging since the Ci is fixed on master

**Pull request merged**